pandas/plotting/_core.py

```diff
@@ -1980,50 +1980,160 @@ def plot_series(data, kind='line', ax=None,                    # Series unique
 
 
 _shared_docs['boxplot'] = """
-    Make a box plot from DataFrame column optionally grouped by some columns or
-    other inputs
+    Make a box plot from DataFrame columns.
+
+    Make a box-and-whisker plot from DataFrame columns optionally grouped
```

Comment:
May be a comma after "columns"?

Reply:
But the first line should be like a short description and then a whole paragraph going into detail, or not?

Reply:
Yes, the comma not in the short summary instead of the period. The comma in the whole paragraph `Make a box-and-whisker plot from DataFrame columns, optionally grouped` :)

Reply:
Great! I'll add that.